worker/sandbox/isolate/sandbox.go

```diff
@@ -19,40 +20,73 @@ import (
 )
 
 var (
-	// The isolate command. Can be overridden with KJUDGE_ISOLATE environment variable.
-	isolateCommand = "isolate"
+	// The isolate command. Can be overridden with KJUDGE_ISOLATE_V1 environment variable.
+	isolateCommandV1     = "isolate"
+	isolateCommandV2     = "isolate"
+	isolateDaemonCommand = []string{"systemctl", "status", "isolate.service"}
 )
 
 func init() {
+	if v, ok := os.LookupEnv("KJUDGE_ISOLATE_V1"); ok {
+		isolateCommandV1 = v
+	}
 	if v, ok := os.LookupEnv("KJUDGE_ISOLATE"); ok {
-		isolateCommand = v
+		isolateCommandV2 = v
+	}
+	if v, ok := os.LookupEnv("KJUDGE_ISOLATE_DAEMON"); ok {
+		isolateDaemonCommand = []string{v}
```

Comment:
This looks suspiciously wrong :thinking: basically this means we should not be able to pass anything with arguments.

I think the better way to do this is to assume `systemd` and only allow customizing the service name.

Reply:
In Docker containers, systemd won't be installed, so KJUDGE_ISOLATE_DAEMON should be able to receive any command. I'm going to make it execute any passed command with `/bin/sh -c`